### DISCUSSION TITLE
Do native builds include some sort of webview bundled in the binary?

### DISCUSSION THREAD

**Discussion by junderw** [DioxusLabs/dioxus#1149] [General]
If I build a crate using dioxus for a native platform, (mobile, desktop etc.) how does dioxus convert the rsx to elements on the screen?

Is it just a small webview with WASM+JS+HTML+CSS? Or does the library convert the VirtualDOM into native UI elements?

I apologize if this has been answered before, a link to more information would suffice. Thanks for your time.

**Comment by DogeDark** [comment]
Currently dioxus-desktop uses Tauri as a webview. There is experimental work on a native WGPU renderer: [Blitz](https://github.com/DioxusLabs/blitz).

At the moment, dioxus-mobile wraps around dioxus-desktop.

**Reply by junderw** [reply]
Thanks!
